scylla/src/frame/response/result.rs

```diff
@@ -589,6 +643,15 @@ fn deser_cql_value(typ: &ColumnType, buf: &mut &[u8]) -> StdResult<CQLValue, Par
             let uuid = uuid::Uuid::from_slice(buf).expect("Deserializing Uuid failed.");
             CQLValue::Uuid(uuid)
         }
+        Varint => {
+            let int_value;
+            if buf[0] == 0x00 {
+                int_value = num_bigint::BigInt::from_bytes_be(Sign::Plus, buf);
+            } else {
+                int_value = num_bigint::BigInt::from_bytes_be(Sign::Minus, buf);
```

Comment:
No changes requested, just a comment: very nice that the serialization format from this crate matches the internal CQL format, makes things much cleaner.